tchannel-core/src/main/java/com/uber/tchannel/api/TChannel.java

```diff
@@ -66,6 +68,8 @@
 
 public final class TChannel {
```

Comment:
this is non-related. but we should remove 'final' , and allow users to override or extend TChannel class.